setup.cfg

```diff
@@ -34,7 +34,7 @@ packages = find_namespace:
 include_package_data = true
 namespace_packages = astronomer,astronomer.providers
 install_requires =
-    apache-airflow>=2.2.0
+    apache-airflow>=2.3.0
```

Comment:
Does this not work with Airflow 2.2.x @pankajkoti @phanikumv -- Bumping the pin means the providers won't be installable for users on 2.2.x which is what we don't want.

Reply:
We observed an issue where some piece of code was calling xcom.execution_date which(the column) does not exist in the upgraded DB for 2.3.0 of our base image.